resources/puppetlabs/lein-ezbake/template/global/tasks/build.rake

```diff
@@ -32,14 +48,82 @@ namespace :pl do
         # carry forward defaults from Packaging::Deb::Repo
         repo = Pkg::Config.apt_repo_name || 'main'
         platform = cow.split('-')[1..-2].join('-')
-        platform_path = "deb/#{platform}/#{repo}"
+
+        # Keep on keepin' on with hardcoded paths in packaging
+        # Hopefully this goes away with packaging 1.0.x.
+        #  --MMR, 2017-08-30
+        if Pkg::Config.build_pe
+          platform_path = "pe/deb/#{platform}"
+        else
+          platform_path = "deb/#{platform}/#{repo}"
```

Comment:
Will `platform` still have dashes here? Don't we want slashes by this point?

Reply:
oh wait, don't we want the codename?

Reply:
With all of the cows we have `platform` is set to the codename
`'base-jessie-i386.cow'split('-')[1..-2].join('-')` = `'jessie'`.  The join should never actually end up inserting a '-' unless the format of the cows change, it's mostly there to convert the array to a string and leave it set to something reasonable that we'd be more likely able to debug in case the format changes

Reply:
ooh, nice. yeah, it was the `join` that made me assume `plat-ver`. 👍